tests/sentry/api/endpoints/test_relay_projectconfigs.py

```diff
@@ -150,6 +151,12 @@ def test_internal_relays_should_receive_full_configs(
     assert safe.get_path(cfg, "config", "datascrubbingSettings", "scrubIpAddresses") is True
     assert safe.get_path(cfg, "config", "datascrubbingSettings", "sensitiveFields") == []
 
+    # Event retention depends on settings, so assert the actual value. Likely
+    # `None` in dev, but must not be missing.
+    assert cfg["config"]["eventRetention"] == quotas.get_event_retention(
```

Comment:
Can we just hardcode the value here? We know what our test settings look like.

Reply:
The hard coded one is None ^^

Reply:
ok can we set the event retention to something and assert it works?

Reply:
Potentially, with a global option. The actual code where this is set lives outside of sentry, so I didn't see a point in this.